packages/framework/block-std/src/scope/block-std-scope.ts

```diff
@@ -15,6 +17,8 @@ export interface BlockStdOptions {
   doc: Doc;
 }
 
+export const services = new Container();
```

Comment:
According to their readme, seems they need decorator stage 2 and reflect metadata. However we're using decorator stage 3 now.

Reply:
https://github.com/inversify/InversifyJS/issues/1507#issuecomment-2066401389

I think we may need to choose another library.